frame/nfts/src/types.rs

```diff
@@ -519,6 +520,8 @@ pub struct PreSignedMint<CollectionId, ItemId, AccountId, Deadline> {
 	pub(super) only_account: Option<AccountId>,
 	/// A deadline for the signature.
 	pub(super) deadline: Deadline,
+	/// An optional price the claimer would need to pay for the mint.
+	pub(super) mint_price: Option<Balance>,
```

Comment:
Why `mint_price` and not just `price`? Isn't it inherent that it is the price for the mint because it is in the `PreSignedMint` struct?